### ISSUE TITLE
Romanized lyrics broken

### ISSUE THREAD

**Issue by TheAbzzz** [surfbryce/beautiful-lyrics#288]
Keep getting this error at the bottom of the app after restarting Spotify. Same error occurs when reinstalling the extension and everything is updated to the latest version. Not themes are applied and using the app directly from the website.
![image](https://github.com/surfbryce/beautiful-lyrics/assets/115082749/6a590227-f310-4936-abe7-5221eb3dc72f)

Almost all the chinese songs I've tested show romanized lyrics like this:
![image](https://github.com/surfbryce/beautiful-lyrics/assets/115082749/a3915a83-54b6-4b27-8fd1-b4ac3d4ce7da)

The japanese songs I've tried appear like this:
![image](https://github.com/surfbryce/beautiful-lyrics/assets/115082749/7dd225db-602b-4441-901a-b847036cf86e)

**Comment by oldschoolvirgo** [comment]
@surfbryce, regarding the first screenshot, spicetify retired .shownotification for [Notistack Snackbar.](https://spicetify.app/docs/development/api-wrapper/modules#snackbar) 

I've had good success using the below code. Haven't yet figured out how to make multi-line notifcations though.

		Spicetify.Snackbar.enqueueSnackbar('Message to User'), {
		  autoHideDuration: 6000,
			variant: 'success',
		}

Great to have you back! Hands down the best lyrics experience out there!

**Comment by surfbryce** [comment]
@TheAbzzz The error you're seeing is not an error and is the update message saying that the Marketplace version got updated to the GitHub version (latest). It just doesn't look right because Spotify messed up some stuff.

I will check out what is happening with Romanization, some stuff did change in this update so it may be related to that.

**Comment by surfbryce** [comment]
@oldschoolvirgo Yeah but shownotification should still properly display the HTML, so Spotify changed something there internally.

It probably would be better to use Snackbar in this instance, apparently I have to use "dangerous" property in React, but the extension doesn't use React so I'll have to find a way to do it in Vanilla JS

**Comment by surfbryce** [comment]
@oldschoolvirgo Ended up using Snackbar and React createElement to keep the HTML working, so it should be fine in the next update when I publish that.

**Comment by surfbryce** [comment]
btw @TheAbzzz can you tell me the songs you used in those images so I could test those specifically? I'll also test out my own

**Comment by TheAbzzz** [comment]
Chinese song was: 
W.M.L x Vicky Chen - I'm Alive

Japanese song was:
KANA-BOON - Silhouette

**Comment by shashankdangi** [comment]
Having the same issue for all the romanization . getting katakana version instead of romaji . 
![image](https://github.com/surfbryce/beautiful-lyrics/assets/91678732/dcc423ee-a945-49f3-ae2d-110bc9668de3)

**Comment by Andrekirb** [comment]
![image](https://github.com/surfbryce/beautiful-lyrics/assets/100699097/0f1882cf-941c-460f-9e6e-43ff04ae52df)
same here
